master/internal/rm/provisioner/aws.go

```diff
@@ -369,6 +369,13 @@ func (c *awsCluster) launchInstances(instanceNum int, dryRun bool) (*ec2.Reserva
 				},
 			},
 		},
+		MetadataOptions: &ec2.InstanceMetadataOptionsRequest{
```

Comment:
we do we have these options only here, and not in `TerminateInstances` and two `DescribeInstances`calls in this file?

Reply:
My understanding is these options are enabled on a per instance basis so the only changes should be where we launch instances (master should have it enabled through the cloudformation changes I believe). 

Once enabled the aws-go-sdk will automatically handle actually using imdsV2 getting the token and setting it as needed without any changes on our side.

Reply:
ah, it's the instance creation parameters, not the extra EC2 API call parameters. got it